modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,32 @@ spec:
 
 |===
 
+.Default protocol for auto-instrumentation
```

Comment:
Pavol, you added it as a [separate section in your draft PR](https://github.com/openshift/openshift-docs/pull/83387/files#diff-e469e721412bda971611ec4d0210f323895fded0c21ab21415c685210e9d3ab2R126), but I'm moving it here into this section as just one of the tables.